ceph-releases/jewel/ubuntu/14.04/daemon/debug.sh

```diff
@@ -11,6 +11,8 @@ for option in $(comma_to_space ${DEBUG}); do
   case $option in
     verbose)
       echo "VERBOSE: activating bash debugging mode."
+      DEBUG_MS=20
+      echo "VERBOSE: Ceph daemon will now run in debugging level $DEBUG_MS"
```

Comment:
What kind of debug options can we pass to CEPH_ARGS? Is that going to help running daemon with a higher debugging level?